app/Http/Requests/User/SendMessageRequest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Http\Requests\User;
+
+use App\Events\User\MessageSent;
+use Illuminate\Foundation\Http\FormRequest;
+
+class SendMessageRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
```

Comment:
authorization here should have a logged in user. or something like that

Reply:
done